sonic-xcvrd/tests/test_xcvrd.py

```diff
@@ -215,3 +215,103 @@ def test_post_port_sfp_dom_info_to_db(self):
+                                                                       'version_slot1_nic': '0.7',
+                                                                       'version_slot2_nic': '0.7',
+                                                                       'run_slot1_nic': '0.7',
+                                                                       'run_slot2_nic': '0.7',
+                                                                       'commit_slot1_nic': '0.7',
+                                                                       'commit_slot2_nic': '0.7',
+                                                                       'empty_slot1_nic': '0.7',
+                                                                       'empty_slot2_nic': '0.7',
+                                                                       'build_slot1_tor1': '0.7',
+                                                                       'build_slot2_tor1': '0.7',
+                                                                       'version_slot1_tor1': '0.7',
+                                                                       'version_slot2_tor1': '0.7',
+                                                                       'run_slot1_tor1': '0.7',
+                                                                       'run_slot2_tor1': '0.7',
+                                                                       'commit_slot1_tor1': '0.7',
+                                                                       'commit_slot2_tor1': '0.7',
+                                                                       'empty_slot1_tor1': '0.7',
+                                                                       'empty_slot2_tor1': '0.7',
+                                                                       'build_slot1_tor2': '0.7',
+                                                                       'build_slot2_tor2': '0.7',
+                                                                       'version_slot1_tor2': '0.7',
+                                                                       'version_slot2_tor2': '0.7',
+                                                                       'run_slot1_tor2': '0.7',
+                                                                       'run_slot2_tor2': '0.7',
+                                                                       'commit_slot1_tor2': '0.7',
+                                                                       'commit_slot2_tor2': '0.7',
+                                                                       'empty_slot1_tor2': '0.7',
+                                                                       'empty_slot2_tor2': '0.7'}))
+    def test_post_port_mux_info_to_db(self):
+        logical_port_name = "Ethernet0"
+        mux_tbl = Table("state_db", "mux_info_tbl")
```

Comment:
Suggest making table names uppercase, as they would be in the real application. (e.g., `STATE_DB`, `MUX_CABLE_INFO`)

Reply:
fixed